src/core/Akka.Cluster/Cluster.cs

```diff
@@ -271,7 +286,16 @@ private static IScheduler CreateScheduler(ActorSystem system)
             return system.Scheduler;
         }
 
-        public void Shutdown()
+        /// <summary>
+        /// INTERNAL API.
+        /// 
+        /// Shuts down all connections to other members, the cluster daemon and the periodic gossip and cleanup tasks.
+        /// Should not called by the user. 
+        /// 
+        /// The user can issue a <see cref="Leave"/> command which will tell the node
+        /// to go through graceful handoff process `LEAVE -> EXITING ->  REMOVED -> SHUTDOWN`.
+        /// </summary>
+        internal void Shutdown()
```

Comment:
Changed the access on this method to `internal` since it's not meant to be directly callable. Technically a breaking API change, but I don't think people were calling this before.